database/pkg.go

```diff
@@ -0,0 +1,72 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	liberr "github.com/konveyor/controller/pkg/error"
+	"github.com/konveyor/controller/pkg/logging"
+	"github.com/konveyor/tackle2-hub/model"
+	"github.com/konveyor/tackle2-hub/settings"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+	"gorm.io/gorm/schema"
+)
+
+var log = logging.WithName("db")
+
+var Settings = &settings.Settings
+
+const (
+	HubDBVersion     = ".hub.db.version"
```

Comment:
Thinking this may not be seeded here.
Also, that the version WRT the migration would not be DB only.